src/main/scala/cromwell/engine/ExecutionStore.scala

```diff
@@ -27,7 +27,12 @@ class ExecutionStore(binding: WdlBinding) {
     for {
       callEntry <- table if callEntry._2 == ExecutionStatus.NotStarted
       call = callEntry._1
-      if call.prerequisiteCalls().forall(table.get(_).get == ExecutionStatus.Done)
+      if call.prerequisiteCalls().forall{
+        case c:Call => table.get(c).get == ExecutionStatus.Done
+        case x =>
+          println(s"runnableCalls: $x")
+          false
```

Comment:
`log`, and I think this should probably be commented as it's not super-obvious what's happening.